api/specs.proto

```diff
@@ -106,8 +106,11 @@ message TaskSpec {
 		ContainerSpec container = 1;
 	}
 
+	// ContainerHostConfigSpec specifies hostconfig for container.
+	ContainerHostConfigSpec hostconfig = 2;
```

Comment:
This breaks the data model. Container related configuration should be part of `ContainerSpec`.